yt/frontends/swift/data_structures.py

```diff
@@ -131,7 +131,9 @@ def _parse_parameter_file(self):
                 self.current_redshift = float(header["Redshift"])
                 # These won't be present if self.cosmological_simulation is false
                 self.omega_lambda = float(parameters["Cosmology:Omega_lambda"])
-                self.omega_matter = float(parameters["Cosmology:Omega_m"])
+                self.omega_matter = float(parameters["Cosmology:Omega_b"]) + float(
+                    parameters["Cosmology:Omega_cdm"]
+                )
```

Comment:
Is this a bug fix or is the `Cosmology:Omega_cdm` key new in some version of SWIFT ? If old versions don't have it, we need to fallback to the old value somehow.